### PR TITLE
Remove almost all of cockroach-prod.

### PR DESCRIPTION
**Pull request by mberhault**
A couple of scripts and the supervisor interface are still there.

### CODE REVIEW

File: Bunchfile [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-github.com/aws/aws-sdk-go/service/elb
-github.com/biogo/store/interval
-github.com/biogo/store/llrb
-github.com/cockroachdb/cockroach/roachpb
-github.com/cockroachdb/cockroach/util
-github.com/cpuguy83/go-md2man/md2man
-github.com/ghemawat/stream
-github.com/gogo/protobuf/proto
-github.com/inconshreveable/mousetrap
-github.com/kolo/xmlrpc
```
Comment (bdarnell) [inline Bunchfile]: Don't we still need some dependency-pinning solution for the xmlrpc library used by the supervisor interface?

Comment (mberhault) [inline Bunchfile]: I'm not sure we do. go get will work just fine, and the tool isn't really needed for anything other than a script or two for the nightly cron jobs. I'd rather just get rid of bunch.

**Comment by tschottdorf** [comment]
Should fix up CI.

File: Bunchfile (hunk 1 above)
Comment (bdarnell) [inline Bunchfile]: `go get` works just fine as long as this library doesn't introduce bugs or backwards-incompatible changes, but we could get spurious failures if they break something. OTOH, there's not much churn there, and are we even using bunch when building this tool for the nightly builds?

**Comment by mberhault** [comment]
I clicked on the "stop building on circleci" which I think should do it.

**Comment by tschottdorf** [comment]
Is there nothing left that benefits from the simply `go test ./...` CircleCI runs by default?

**Comment by mberhault** [comment]
nope, just the supervisor thing which doesn't have tests. It should probably go away in the long term anyway, it's here just to avoid the extra dependency on xmlrpc from the cockroach repo.

**Comment by tschottdorf** [comment]
Ok. Building it would still make sense but I won't heckle for it.

**Comment by bdarnell** [comment]
If that's the case then I'd be fine with moving the supervisor tool over to the main repo. I wanted to keep all the aws/google/docker/etc client libraries out of the main repo, but if it's just xmlrpc then it's not worth the trouble to keep it separate. But that's another PR; this one LGTM

**Pull request merged**